containerd-shim-v2/service.go

```diff
@@ -556,8 +571,10 @@ func (s *service) Exec(ctx context.Context, r *taskAPI.ExecProcessRequest) (_ *p
 
 // ResizePty of a process
 func (s *service) ResizePty(ctx context.Context, r *taskAPI.ResizePtyRequest) (_ *ptypes.Empty, err error) {
-	span, _ := trace(s.ctx, "ResizePty")
-	defer span.Finish()
+	if os.Getppid() == 1 {
+		span, _ := trace(s.ctx, "ResizePty")
+		defer span.Finish()
+	}
```

Comment:
@cmaf Are the ppid checks useful though outside of New(), Shutdown(), Create() and Delete()?  I would assume that the first fork exits after spawning the second one...

Reply:
@pmores you're right, they actually don't need to be in all functions. However I am removing all the checks because we decided to trace all functions irrespective of ppid.